app/application/PMTResponse3inchR12199_02.cc

```diff
@@ -36,13 +39,16 @@ void PMTResponse3inchR12199_02::Initialize(int seed, const string &pmtname)
         }
     }
     Configuration *Conf = Configuration::GetInstance();
+    Conf->GetValue<float>(s["TimingResConstant"], fTResConstant);
+    Conf->GetValue<float>(s["TimingResMinimum"], fTResMinimum);
     Conf->GetValue<float>(s["ScalFactorTTS"], fSclFacTTS);
     Conf->GetValue<string>(s["SPECDFFile"], fTxtFileSPECDF);
     this->LoadCDFOfSPE(fTxtFileSPECDF);
 }
 
 float PMTResponse3inchR12199_02::HitTimeSmearing(float Q)
 {
+    Q = (Q > 0.5) ? Q : 0.5;
```

Comment:
why?

Reply:
It is from https://github.com/WCTE/WCSim/blob/nuPRISM/develop/src/WCSimWCDigitizer.cc#L296, to avoid blowing up the `timingResolution` variable below